ctapipe/utils/astro.py

```diff
@@ -0,0 +1,55 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from astropy.coordinates import Angle
+from astropy.coordinates import SkyCoord
+from astropy import units as u
+import logging
+
+logger = logging.getLogger(__name__)
+
+__all__ = ['get_bright_stars']
+
+
+def get_bright_stars(pointing=SkyCoord(ra=0. * u.rad, dec=0. * u.rad, frame='icrs'),
+                     radius=180. * u.deg, magnitude_cut=7.96):
```

Comment:
Maybe use a radius that is closer to the max. fov of CTA telescope's? Maybe 15 degrees? Agreed with @kosack on the magnitude cut.  use `None` there.

Reply:
So the default is `None` now, which loads the whole catalog. Advantage of this standard I see is that loading the full sky (i.e., radius 180°), doesn't need a (standard) pointing to be defined. But please correct me if you see a more convenient standard.

Reply:
Mmh okay, so this is a case where the `pointing = None` makes sense. 
Maybe we should split it into two methods, one which loads the catalog and one which searches around a position in the sky?

Reply:
Yes, I was also thinking about this, especially if the funcion is used in a loop. But then you could also make a more simple function loading just the catalog, and the user has to code the two lines to cut it her/himself. As it is now I thought it's a convenience feature to allow to load & select in region/magnitude in just one line...